backend/src/main/java/com/alibaba/higress/console/controller/dto/Domain.java

```diff
@@ -38,4 +39,15 @@ public static class EnableHttps {
     private String enableHttps;
 
     private String certIdentifier;
+
+    public boolean valid() {
+        if (StringUtils.isAnyBlank(name, version)) {
+            return false;
+        }
+        if (StringUtils.isNotEmpty(enableHttps)) {
+            return EnableHttps.OFF.equals(enableHttps) || EnableHttps.ON.equals(enableHttps)
```

Comment:
If new fields are added to this class, is it still OK to return here?
And shall we check certificateId if user enables HTTPS?

Reply:
1. You can have a try with the `enableHttps` change.
2. Maybe performing validation in the service is a better idea.